src/login/styles/_Nins.scss

```diff
@@ -30,6 +30,18 @@
       display: flex;
       justify-content: space-between;
     } 
+    & .link {
+      margin: 0.5rem 0;
+      & a span:nth-child(1){
```

Comment:
is the nth-child necessary? I seem to get the same styling without it. Are there other places where we have more than one link + span in a .vetting-button?